lib/ntbb-session.lib.php

```diff
@@ -471,7 +471,12 @@ function getAssertion(string $userid, string $serverhostname, array $user = null
 		if (($user['userid'] === $userid) && !empty($user['loggedin'])) {
 			// already logged in
 			$usertype = '2';
-			if (in_array($userid, $psconfig['sysops'], true)) {
+			$ismain = $serverhostname === 'sim3.psim.us' || $serverhostname === 'smogtours.psim.us';
+			if (
+				in_array($userid, $psconfig['sysops'], true) ||
+				// give admins and leaders sysop on side servers
```

Comment:
The [last Upper Staff discussion](https://discord.com/channels/534298043176976395/565617827298541572/942897233207128124) on sysop for side servers concluded that it wasn't worth it, since side servers could still disable it and Main leadership shouldn't be in charge of moderating side servers anyway